arangod/Aql/MaterializeExecutor.h

```diff
@@ -120,6 +120,8 @@ class MaterializeRocksDBExecutor : public MaterializeExecutorBase {
 
  private:
   PhysicalCollection* _collection{};
+  std::vector<LocalDocumentId> _docIds;
+  std::vector<InputAqlItemRow> _inputRows;
```

Comment:
There is no reason to have more than 32 keys in the MultiGet call.
You can look MultiGet wrapper used for search